pkg/reset/rdsbackup.go

```diff
@@ -80,8 +80,8 @@ func DeleteRdsBackups(rdsService RdsService) error {
 				DbiResourceId: dbInstanceBackup.DbiResourceId,
 			}
 
-			// Ignoring this output, since changing retention period to 0 takes care of deleting automated backups.
-			rdsService.DeleteDBInstanceAutomatedBackup(deleteDBInstanceAutomatedBackupInput)
+			// Ignoring errors, since changing retention period to 0 takes care of deleting automated backups.
```

Comment:
From the comment here, I'm gathering that we're explicitly ignore errors from this method?
Is that correct @marinatedpork , @AmudaPalani ?